batch/eeprom/DD2090_R4M1E4_oak_d_ffc_4p.json

```diff
@@ -0,0 +1,12 @@
+{
+    "batchName": "",
+    "batchTime": 0,
+    "boardConf": "IR-C00M00-00",
+    "boardName": "DD2090",
+    "boardRev": "R4M1E4",
+    "productName": "OAK-D FFC-4P",
+    "boardCustom": "",
+    "hardwareConf": "F0-FV00-BC000",
+    "boardOptions": 3,
```

Comment:
boardOptions: 0

for USB based devices. (no modifications for PCIe clock and no modifications for USB3 phy clock)

Reply:
@themarpe updated boardOptions, I don't show the camera streams for FFC-4P at all at @Luxonis-David's suggestion (it was also easier to implement), I could add that option (to search for cameras and show the stream when available without dropping the test if not) in a later PR